lms/views/dashboard/api/assignment.py

```diff
@@ -21,6 +22,15 @@ class ListAssignmentsSchema(PaginationParametersMixin):
     """Return assignments that belong to the course with this ID."""
 
 
+class AssignmentsMetricsSchema(PyramidRequestSchema):
+    """Query parameters to fetch metrics for assignments."""
+
+    location = "querystring"
+
+    h_userids = fields.List(fields.Str())
+    """Return metrics for these users only."""
```

Comment:
I commented in another PR about adding some validation to the format of items in this list.

Reply:
Created a note about this: https://github.com/orgs/hypothesis/projects/135?pane=issue&itemId=70312222

Should be pretty easy to implement and apply it to all of these.